### PR TITLE
Rename SV column names to SV ids.

### PR DESCRIPTION
**Pull request by keyurva**
* Will set sv id == column name for id compatible column names in the next PR.

### CODE REVIEW

File: simple/sample/output/process/debug_resolve_powerplants.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 input,dcid,link
-FOO BAR,*UNRESOLVED*,
```
Comment (shifucun) [inline simple/sample/output/process/debug_resolve_powerplants.csv:2]: this file seems to change every time, is there a way to make it stable?

File: simple/sample/output/process/report.json [hunk 2]
```diff
@@ -1,37 +1,37 @@
 {
   "status": "SUCCESS",
-  "startTime": "2023-10-23 10:50:22.644858",
-  "lastUpdate": "2023-10-23 10:50:30.534262",
+  "startTime": "2023-10-23 14:43:10.816166",
```
Comment (shifucun) [inline simple/sample/output/process/report.json:3]: shall we gitignore this file?

File: simple/stats/nodes.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    self.variables[sv_column_name] = StatVar(
+        f"{_CUSTOM_SV_ID_PREFIX}{len(self.variables) + 1}",
+        var_cfg.name,
+        description=var_cfg.description,
+        nl_sentences=var_cfg.nl_sentences,
+        group_id=group_id,
+    )
+    return self.variables[sv_column_name]
+
+  def group(self, group_path: str) -> StatVarGroup | None:
```
Comment (shifucun) [inline simple/stats/nodes.py:55]: just thinking ahead, if we use column name as dcid, then the group path might to have the stat var name. for exampele:

"country_risk": "Floret/Country_Risk"

**Review by shifucun — COMMENTED** [review]
nice simplification!

File: simple/sample/output/process/debug_resolve_powerplants.csv (hunk 1 above)
Comment (keyurva) [reply simple/sample/output/process/debug_resolve_powerplants.csv:2]: Yeah it's annoying. Will look into it after the next PR.

File: simple/sample/output/process/report.json (hunk 2 above)
Comment (keyurva) [reply simple/sample/output/process/report.json:3]: Let's keep it around for now until we have reached a steady state.

One way we can address it is to have a mode that persists it with fixed timestamps for samples and tests.

File: simple/stats/nodes.py (hunk 3 above)
Comment (keyurva) [reply simple/stats/nodes.py:55]: Group paths don't have stat var names. In the above case, the group name will only be "Floret" and the group ID is generated by us (e.g. "custom/g/group_1")

Comment (shifucun) [reply simple/stats/nodes.py:55]: Oh, now i remember, there is a separate name field for the stat var.

**Review by shifucun — APPROVED** [review]

**Pull request merged**